.eslintignore

```diff
@@ -0,0 +1,10 @@
+**.md
+**.html
+**.css
+.gitignore
+css/**
+img/**
+js/jquery.2.2.0.min.js
```

Comment:
You've got a typo in the jQuery file name, so it comes up with warnings in eslint.